dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -5384,6 +5384,13 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey USER_CLIENT_CACHE_FALLBACK_ENABLED =
+      booleanBuilder(Name.USER_CLIENT_CACHE_FALLBACK_ENABLED)
+          .setDefaultValue(false)
+          .setDescription("If this is enabled, local cache will fallback to external storage.")
```

Comment:
We shall highlight that, this fallback is on errors in addition to normal fallback on cache miss.
Otherwise, falling back to external storage sounds trivial to me due to cache misses

```suggestion
          .setDescription(""When enabled, reading from local cache will fall back to "
              + "external storage in case of any local errors in addition to cache misses.")
```